### PR TITLE
remove nothing found from different template

### LINKED ISSUE #2005: Minimize duplication of code in templates
**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by shubham3211**
Fixes #2005 
Sample:- https://shubham3211.github.io/filter-remove/

### CODE REVIEW

**Comment by shubham3211** [comment]
@agbilotia1998 @Kreijstal please review.

File: src/backend/templates/rooms.hbs [hunk 1]
```diff
@@ -470,12 +470,7 @@
           var temp = JSON.parse(localStorage[eventName]);
           main.trackRoomFilter('room', trackFilterMode, roomFilterMode, trackName, roomName, temp, mode);
           // show "no results" if search input value matches zero items
-          if ($('.date-filter:visible').length == 0) {
-            $('#no-results').remove();
-            $('.search-filter:first').after('<p id="no-results" style="color: red">No matching results found.</p>');
-          } else {
-            $('#no-results').remove();
-          }
+          main.showNoResult('room');
```
Comment (agbilotia1998) [inline src/backend/templates/rooms.hbs]: @shubham3211 I guess you should add here both the arguments.

**Comment by shubham3211** [comment]
@agbilotia1998 I have made the changes.

**Comment by agbilotia1998** [comment]
@shubham3211 I guess tests are not failing due to this PR. I am restarting the build. Thanks :)

**Comment by shubham3211** [comment]
@agbilotia1998 please review

**Pull request merged**